### PR TITLE
Update client package locks to reflect server release

### PR DESCRIPTION
**Pull request by heliocliu**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -3025,40 +3019,13 @@
       "dev": true
     },
     "conventional-changelog-angular": {
-      "version": "5.0.11",
-      "resolved": "https://registry.npmjs.org/conventional-changelog-angular/-/conventional-changelog-angular-5.0.11.tgz",
-      "integrity": "sha512-nSLypht/1yEflhuTogC03i7DX7sOrXGsRn14g131Potqi6cbGbGEE9PSDEHKldabB6N76HiSyw9Ph+kLmC04Qw==",
+      "version": "5.0.10",
```
Comment (heliocliu) [inline package-lock.json:3022]: why are these going backwards? that seems not right

Comment (curtisman) [reply package-lock.json:3022]: clean your node_modules first?

**Review by curtisman — APPROVED** [review]
:shipit:

**Comment by heliocliu** [comment]
something went wrong with this branch.  retrying in #3684

**Pull request closed without merging**